docs/api-reference/mapbox/mapbox-layer.md

```diff
@@ -89,6 +91,86 @@ map.on('load', () => {
 }
 ```
 
+#### React
+
+```js
+import React from "react";
+import DeckGL from "@deck.gl/react";
+import { ScatterplotLayer } from "@deck.gl/layers";
+import { MapboxLayer } from "@deck.gl/mapbox";
+import { StaticMap } from "react-map-gl";
+
+const initialViewState = {
+  longitude: -112.1861,
+  latitude: 36.1284,
+  zoom: 12.1,
+  pitch: 0,
+  bearing: 0,
+};
+
+export default class Map extends React.Component {
+  state = {};
+
+  // DeckGL and mapbox will both draw into this WebGL context
+  _onWebGLInitialized = (gl) => {
+    this.setState({ gl });
+  };
+
+  _onMapLoad = () => {
+    const map = this._map;
+    const deck = this._deck;
+
+    // You must initialize an empty deck.gl layer to prevent flashing
+    map.addLayer(
+      // This id has to match the id of the deck.gl layer
+      new MapboxLayer({ id: "my-scatterplot", deck }),
+      // Optionally define id from Mapbox layer stack under which to add deck layer
+      'beforeId'
```

Comment:
Probably double quotes for consistency

Reply:
There's already some quote inconsistency across examples. Earlier examples use single quotes. Maybe I should update all quotes for all examples to be either single quotes or double quotes.